index/secondary_index_manager.cc

```diff
@@ -347,7 +347,7 @@ bool secondary_index_manager::is_global_index(const schema& s) const {
     });
 }
 
-std::optional<sstring> secondary_index_manager::vector_index_class(const schema& s) const {
+std::optional<sstring> secondary_index_manager::custom_index_class(const schema& s) const {
```

Comment:
Why not name this function `custom_index_class` in the commit where you add this function in the first place? In general, you should avoid changes like this, i.e. doing something in one commit and then "fixing" it in another place - if you can introduce the thing "correctly" in the first place.